fossunited/stack/utils.py

```diff
@@ -1,15 +1,14 @@
+from collections import defaultdict
+
 import frappe
 
 
 def get_stack_dict():
     stacks = frappe.get_all(doctype="Stack", fields=["name1", "icon", "category"], page_length=999)
 
-    stack_dict = {}
+    stack_dict = defaultdict(list)
 
     for item in stacks:
-        if item.category not in stack_dict:
-            stack_dict.update({item.category: []})
-
         stack_dict[item.category].append(item)
 
-    return stack_dict
+    return dict(stack_dict)
```

Comment:
q: i'm not sure if you have to do `dict(stack_dict)` because `defaultdict` already implement the relevant interface if I remember correctly.

Reply:
I think either is fine, I am going ahead and removing the `dict` as it works the same